apps/recnet-api/src/modules/rec/rec.service.ts

```diff
@@ -39,10 +39,12 @@ export class RecService {
   public async getRecs(
     page: number,
     pageSize: number,
-    userId: string
+    userId: string,
+    excludeCutoff: Date
```

Comment:
```suggestion
    to: Date
```

I am thinking something like `to: Date`, so that we can get all recs before `to`. I felt like this make this function easier to understand and use. (If use `excludeCutoff` the date here need to strictly match the "cutoff" time in a week) What do you think?

Reply:
Great suggestion. I think it will be clearer and easier to understand for users if we apply this change in the service layer. Yet I will stick to the original naming in the RecFilter to avoid logic leaking to repository layer.

Reply:
Good suggestion!
I have slightly changed the filter logic to apply `to`. Please take a look, thx!